spdm.rflx

```diff
@@ -0,0 +1,1487 @@
+            then null;
+         Error_Response_1_0 : Error_Response_1_0
+            then null;
+         Error_Response_1_1 : Error_Response_1_1
+            then null;
+         Key_Exchange_Response : Key_Exchange_Response
+            then null;
+         Finish_Response : Finish_Response
+            then null;
+         Key_Update_ACK_Response : Key_Update_Ack_Response
+            then null;
+         End_Session_Response : End_Session_Response;
+      end message;
+
+   --  FIXME: Update transcript hash
+
+   generic
+      Transport : Channel with Readable, Writable;
+   session Responder with
+      Initial => Init,
+      Final => End_Session
+   is
+      Request : SPDM::Request;
+      Supported_Versions : Version_Number_Entries;
+   begin
+      state Init
+      is
+      begin
+         Supported_Versions'Append
+            (Version_Number_Entry'(Alpha                 => 0,
+                                   Update_Version_Number => 0,
+                                   Minor_Version         => 1,
+                                   Major_Version         => 1));
+         --  FIXME: Reset all state
+         Transport'Read (Request);
+      transition
+         goto Version
+            --  In case a Request is not valid the exception transition
+            --  should be chosen but this is not implemented yet.
+            --  ISSUE: Componolit/RecordFlux#691
+            if Request'Valid and Request.Code = Get_Version
+         goto Caps
+            --  ISSUE: Componolit/RecordFlux#691
+            if Request'Valid and Request.Code = Get_Capabilities
+         goto Error_Unsupported_Request
+            if Request'Valid
+         goto Error_Invalid_Request
+      exception
+         goto Error_Unspecified
+      end Init;
+
+      state Version
+      is
+      begin
+         Transport'Write
+            (Response'(Major_Version                               => 1,
+                       Minor_Version                               => 0,
+                       Code                                        => Version,
+                       Version_Response_Param_1                    => 0,
+                       Version_Response_Param_2                    => 0,
+                       Version_Response_Reserved                   => 0,
+                       Version_Response_Version_Number_Entry_Count => 1,
+                       Version_Response_Version_Number_Entries     => Supported_Versions));
+         Transport'Read (Request);
+      transition
+         goto Version
+            --  ISSUE: Componolit/RecordFlux#691
+            if Request'Valid and Request.Code = Get_Version
+         goto Caps
+            --  ISSUE: Componolit/RecordFlux#691
+            if Request'Valid and Request.Code = Get_Capabilities
+         goto Error_Unsupported_Request
+            if Request'Valid
+         goto Error_Invalid_Request
+      exception
+         goto Error_Unspecified
+      end Version;
+
+      state Caps
```

Comment:
Maybe we should rename that into `Capabilities`.

Reply:
Fixed in https://github.com/Componolit/RecordFlux-specifications/pull/93/commits/7c1908a0db7eece9cbbb987fa14b0c9d9e4ec91f.